docusaurus/docs/welcome/instructions/app-icon.md

```diff
@@ -45,24 +73,26 @@ Note that the filenames of the generated assets should be renamed according to t
 
 #### Android
 
-After moving the assets to the following locations, there is nothing more to do.
-Both "Square" and "Circle" shapes should work.
+> 💡 Note that if you used [Image Asset Studio](https://developer.android.com/studio/write/image-asset-studio), you won't have to bother about this, because the tool automatically places the assets into the correct locations.
```

Comment:
There is no need to link Image Asset Studio again, I guess.

```suggestion
> 💡 Note that if you use Image Asset Studio, you won't have to bother about this, because the tool automatically places the assets into the correct locations.
```